drivers/firmware/dmi_scan.c

```diff
@@ -166,6 +166,7 @@ static int __init dmi_checksum(const u8 *buf, u8 len)
 static const char *dmi_ident[DMI_STRING_MAX];
 static LIST_HEAD(dmi_devices);
 int dmi_available;
+EXPORT_SYMBOL_GPL(dmi_available);
```

Comment:
I'm curious what maintainers will have to say about this... As an alternative we could call `dmi_walk()` with a dummy callback and check for `-ENXIO` but that would be rather ugly I admit...

Reply:
there were no objections, see alsa-devel mailing list.